docs/wiki/Deploying-Enterprise-Scale-HubAndSpoke.md

```diff
@@ -1,12 +1,10 @@
-## In this Section
+## Enterprise-Scale deployment with hub and spoke network topology
 
-TBD: Enterprise Scale Deployment Accelerator is a bla, bla, bla. In this quickstart, you will implement an Azure foundation that provides your Landing Zones with connectivity on-premises datacenters and branch offices by leveraging a hub and spoke network topology.
-
-Please refer to [AdventureWorks reference implementation](https://github.com/Azure/Enterprise-Scale/blob/main/docs/reference/adventureworks/README.md) for further details on the Azure foundation enabled in this tutorial.
+This section will describe how to deploy an Enterprise-Scale Landing Zones platform with connectivity to on-premises datacenters and branch offices based on a hub and spoke network topology. Please refer to [AdventureWorks reference implementation](https://github.com/Azure/Enterprise-Scale/blob/main/docs/reference/adventureworks/README.md) for further details on this reference architecture.
```

Comment:
Shouldnt this be introducing the "azure landing zones deployment accelerator" basic flow?